mlflow/models/evaluation/base.py

```diff
@@ -122,6 +122,7 @@ def __init__(
         version=None,
         metric_details=None,
         metric_metadata=None,
+        custom_metric_config=None,
     ):
```

Comment:
Can we add a docstring for the new argument?

Reply:
oh good catch, still not used to docstring stuff.